specifications/design-notes/tx-resubmission.md

```diff
@@ -0,0 +1,56 @@
+# Transaction resubmission
+
+Any transaction on the Cardano blockchain can be rolled back due to a
+chain switch, until its block is _2k_ blocks deep.
```

Comment:
~2k~ → `3k/f`  

2k was true in Ouroboros Classic / BFT but not in Praos.

Reply:
Doh forgot, thanks.